gitoxide-core/src/organize.rs

```diff
@@ -177,7 +177,11 @@ fn handle(
             match kind {
                 RepoKind::Bare => path,
                 RepoKind::WorkingTree => {
-                    path.set_extension("");
+                    if let Some(ext) = path.extension() {
+                        if ext == "git" {
```

Comment:
This would also work, but I prefer the existing semantics.
```rust
if path.extension().map_or_else(|| false, |e| e == "git") {
    path.set_extension("");
}
```

Reply:
Sometimes plain and simple is the easiest to read, indeed.